facades/forecast.js

```diff
@@ -0,0 +1,31 @@
+var User = require('../models').User;
+var GeocodeService = require('../services/geocode');
+var ForecastService = require('../services/forecast');
+
+module.exports = class ForecastFacade {
+  constructor(status, body) {
+    this.status = status
+    this.body = body
+  }
+
+  static forecastForUser(api_key, location) {
+    console.log(location);
+    return new Promise((resolve, reject) => {
+      User.authenticate(api_key)
+      .then(function() {
+        return GeocodeService.requestLocation(location)
+      })
+      .then(function(location) {
+        console.log(location);
+        return ForecastService.requestForecast(location)
+      })
+      .then(function(forecast) {
+        delete forecast.minutely;
```

Comment:
@plapicola why did you need to delete the minutely? Did you choose that over creating logic in the forecast for whether or not you needed the minutely?

Reply:
I still need to format the request to more closely match the spec, but for an initial pass just opted to delete the minutely information as this was not requested in the spec. There are additional nested keys I'll need to work over to resolve. A separate class might be helpful in achieving this. I'd love your thoughts.